### PR TITLE
Changes to fetch and display orders and OTEL instrumentation

### PR DESCRIPTION
**Pull request by renukakrishnan**
*Description of changes:*
1. Added server side and UI code to fetch and display the orders
2. Added AWS OTEL instrumentation for metrics and traces

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: applications/dotnet/src/WebUI/otel-collector-config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http: null
+processors:
+  batch: null
+exporters:
+  prometheusremotewrite:
+    endpoint: <YOUR-APS-WORKSPACE-ENDPOINT>/api/v1/remote_write
```
Comment (shapirov103) [inline applications/dotnet/src/WebUI/otel-collector-config.yml:11]: Is this correct config? Who is going to populate the endpoint?

Comment (shapirov103) [reply applications/dotnet/src/WebUI/otel-collector-config.yml:11]: @renukakrishnan ^^

Comment (shapirov103) [reply applications/dotnet/src/WebUI/otel-collector-config.yml:11]: @elamaran11 I assume we will have an approach that enables us to scape metrics from the app without spreading the knowledge of the remote write endpoint. It is not going to scale and requires changes between dev and prod. 

@renukakrishnan please check if there is an approach to emit OTEL metrics from the app in a way that can be scraped by the OTEL collector.

Comment (elamaran11) [reply applications/dotnet/src/WebUI/otel-collector-config.yml:11]: @shapirov103 Yes Dev team should not be doing this, we have already wired collector config from Platform side on both Dev and Prod clusters to send metrics to AMP. Yes Dev team doing at app leve is not scalable approach.

Comment (renukakrishnan) [reply applications/dotnet/src/WebUI/otel-collector-config.yml:11]: @shapirov103 I have added the internal DNS name of the OTEL collector. Also attaching a file that shows the metrics that will get emitted.[
[prometheus_exporter_data.txt](https://github.com/user-attachments/files/17282756/prometheus_exporter_data.txt)
](url)

File: applications/dotnet/src/Persistence/DependencyInjection.cs [hunk 2]
```diff
@@ -9,8 +9,23 @@ public static class DependencyInjection
     {
         public static IServiceCollection AddPersistence(this IServiceCollection services, IConfiguration configuration)
         {
+            var configurationbuilder = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .AddJsonFile($"appsettings.Local.json", optional: true)
+                .AddKeyPerFile("/opt/secret-volume", optional: true, reloadOnChange: true)
```
Comment (shapirov103) [inline applications/dotnet/src/Persistence/DependencyInjection.cs:18]: Nice!

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**